README.md

```diff
@@ -479,6 +509,89 @@ registry.define('my-widget', () => {
 });
 ```
 
+#### Injecting State
+
+Working with larger widget structures, it can quickly become tiresome and complex to pass all the required properties down the tree. Additionally having to do so often means that widgets are forced to be aware of properties that they they don't need, but are required to know about in order to ensure they are propagated to the children.
```

Comment:
Maybe in the `w` section? Something like:

```ts
[in Baz.ts]

protected render: DNode {
  return v(div, {}, [
    w(Foo, {}, [Qux]);
    w(Bar, {});
  ]);
}
```
So now you have created instances of `Foo` and `Bar` widgets from inside the `render` call of `Baz` widget. Because you passed a child `Qux` to the `w` call of `Foo`, you have created a tree of widgets like so:

```
div (Baz)
  |
  - Foo
  |   |
  |   - Qux
  |
  - Bar
```
Note that in the DOM, you will see the result of the `render` call for `Foo, Bar, and Qux`;